met-api/src/met_api/services/comment_service.py

```diff
@@ -0,0 +1,82 @@
+
+"""Service for comment management."""
+from met_api.models.comment import Comment
+from met_api.schemas.comment import CommentSchema
+from met_api.schemas.submission import SubmissionSchema
+from met_api.schemas.survey import SurveySchema
+
+
+class CommentService:
+    """Comment management service."""
+
+    otherdateformat = '%Y-%m-%d'
+
+    @staticmethod
+    def get_comment(comment_id) -> CommentSchema:
+        """Get Comment by the id."""
+        comment = Comment.get_comment(comment_id)
+        comment_schema = CommentSchema()
+        return comment_schema.dump(comment)
+
+    @classmethod
+    def get_comments_by_survey_id(cls, user_id, survey_id):
+        """Get all comments."""
+        if not user_id:
+            comment_schema = CommentSchema(many=True, only=('text', 'submission_date', 'survey'))
+            return comment_schema.dump(Comment.get_publicly_viewable_comments_by_survey_id_query(survey_id))
+
+        comment_schema = CommentSchema(many=True)
+        return comment_schema.dump(Comment.get_comments_by_survey_id_query(survey_id))
+
+    @classmethod
+    def create_comments(cls, comments: list):
+        """Create comment."""
+        for comment in comments:
+            cls.validate_fields(comment)
+
+        return Comment.bulk_create_comment(comments)
```

Comment:
the word bulk is bulky ..lol..
may be save_comments is easier?
there is an sql alchmey real bulk command which we are not using..
ok not to change this as well..

Reply:
changed to add_all_comments which matches db.session.add_all(new_comments)